sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -386,6 +511,37 @@ constant property.
 
 |===
 
+Free functions of the following form are defined for the overloaded operators for
```

Comment:
If you aren't going to list the overloaded operators I think you at least need to say:
```suggestion
Free functions of the following form are defined for the overloaded operators listed above for
```